### DISCUSSION TITLE
Errors in running notebook number 2

### DISCUSSION THREAD

**Discussion by BobbieZHU** [joaofig/ved-explore#10] [General]
Hi!

When I am running the cell:

for file in tqdm(files):
    df = read_data_frame(file)
    
    signals = []
    for row in df.itertuples(index=False):
        signals.append(row)
        
    db.insert_signals(signals)

There is always a error named: OperationalError: no such table: signal. I uploaded the detail explanation on the attachment.
<img width="860" alt="bc40680efd69d80cf68f38ff82a837a" src="https://user-images.githubusercontent.com/88154436/127525789-00667f7e-df30-499a-942b-45ace68a89c9.png">

I have difficulty in solving this problem. Could you please give me some advice?  

Thanks，
Bobbie

**Comment by BobbieZHU** [comment]
By the way, I am using the Jupyter embedded in Anaconda.

**Comment by joaofig** [comment]
You are right. I am missing an update to the code that I will do very soon. I will let you know when it is ready. Thank you for letting me know!
